packages/frontend/pages/index.tsx

```diff
@@ -34,7 +55,9 @@ const Home: NextPage = () => {
         <Address address="0xd8da6bf26964af9d7eed9e03e53415d37aa96045" />
 
         <p>
-          <button className="btn btn-primary">Daisy UI Button</button>
+          <button className="btn btn-primary" onClick={() => tempTest.onClick()}>
```

Comment:
The tx with the text contract doesn't seem to be working for me. What do I need to do?

![image](https://user-images.githubusercontent.com/2486142/194786105-33918cb6-de3c-4761-8eb4-d11a938a08d6.png)

The unknown account is my burner wallet address

Reply:
you have a local chain running?

Reply:
hmm i'll look into this

Reply:
i pushed some changes @carletex  There was a bug.
Also a reminder, before it works, it needs yarn chain, and eth for the burner account.

Reply:
Ok, now it works